httpapi/coreapi/types.go

```diff
@@ -321,11 +319,12 @@ func toNFTMintRequest(req MintNFTRequest, registryAddress common.Address) nft.Mi
 	return nft.MintNFTRequest{
 		DocumentID:               req.DocumentID,
 		DepositAddress:           req.DepositAddress,
-		GrantNFTReadAccess:       req.GrantNFTReadAccess,
+		GrantNFTReadAccess:       false,
 		ProofFields:              req.ProofFields,
 		RegistryAddress:          registryAddress,
-		SubmitNFTReadAccessProof: req.SubmitNFTReadAccessProof,
-		SubmitTokenProof:         req.SubmitTokenProof,
+		AssetManagerAddress:      common.HexToAddress(req.AssetManagerAddress.String()),
+		SubmitNFTReadAccessProof: false,
+		SubmitTokenProof:         true,
```

Comment:
We can remove this as well IMO

Reply:
@vedhavyas I can do that. I will default to these values in the service code itself, and delete from here the flags. Is that what you meant?

Reply:
resolved